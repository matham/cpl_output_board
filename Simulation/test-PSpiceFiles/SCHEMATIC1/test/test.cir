** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
.INC "C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1\test\test_profile.inc" 
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Matthew Einhorn\Documents\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.AC DEC 1000 0.01 10Meg
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
