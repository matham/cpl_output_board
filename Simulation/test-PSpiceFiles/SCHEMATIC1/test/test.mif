 lib=C:\OrCAD_16.6_Lite\tools\PSpice\Library\opamp.lib, offset=49754, size=419
 lib=C:\OrCAD_16.6_Lite\tools\PSpice\Library\opamp.lib, offset=36624, size=1928
