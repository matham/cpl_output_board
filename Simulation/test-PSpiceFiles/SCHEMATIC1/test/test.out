
**** 01/24/13 23:13:17 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
.INC "C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1\test\test_profile.inc" 
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\Matthew Einhorn\Documents\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:

**** INCLUDING test_profile.inc ****
.LIB    ".\opamp.lib" 

**** RESUMING test.cir ****
.lib "nomd.lib" 

*Analysis directives: 
.AC DEC 1000 0.01 10Meg
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
V_V1         N00146 0 10Vdc
V_V2         N00116 0 -10Vdc
V_V3         N00334 0 DC 1Vdc AC .5Vac 
R_R1         N00247 N00208  7.25k  
R_R2         0 N00247  2.5k  
C_C1         N00334 N00366  4.7u  
C_C2         N00366 N00370  4.7u  
R_R3         0 N00366  12.5k  
R_R4         N00208 N00366  10.2k  
X_U3A         N00370 N00247 N00146 N00116 N00208 LM258

**** RESUMING test.cir ****
.END

**** 01/24/13 23:13:17 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U3A.x_lm258.dx 
          IS  800.000000E-18 
          RS    1            


               X_U3A.x_lm258.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 01/24/13 23:13:17 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U3A.x_lm258.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 01/24/13 23:13:17 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1


 ****     SMALL SIGNAL BIAS SOLUTION       TEMPERATURE =   27.000 DEG C


******************************************************************************



 NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE     NODE   VOLTAGE


(N00116)  -10.0000 (N00146)   10.0000 (N00208)    9.3154 (N00247)    2.3887     

(N00334)    1.0000 (N00366)    5.1296 (N00370)    2.6452                        

(X_U3A.x_lm258.6)   -1.4736           (X_U3A.x_lm258.7)   11.3460               

(X_U3A.x_lm258.8)   11.3460           (X_U3A.x_lm258.9)    0.0000               

(X_U3A.x_lm258.10)    3.0275          (X_U3A.x_lm258.11)   -9.9216              

(X_U3A.x_lm258.12)  -10.0000          (X_U3A.x_lm258.13)    3.0001              

(X_U3A.x_lm258.14)    3.0275          (X_U3A.x_lm258.53)    8.5000              

(X_U3A.x_lm258.54)   -9.3500          (X_U3A.x_lm258.90)   40.6090              

(X_U3A.x_lm258.91)   40.0000          (X_U3A.x_lm258.92)  -40.0000              

(X_U3A.x_lm258.99)    0.0000          




    VOLTAGE SOURCE CURRENTS
    NAME         CURRENT

    V_V1         3.703E-02
    V_V2         2.217E-03
    V_V3         0.000E+00
    X_U3A.x_lm258.vb  -1.474E-05
    X_U3A.x_lm258.vc  -3.924E-02
    X_U3A.x_lm258.ve   1.898E-11
    X_U3A.x_lm258.vlim   4.061E-02
    X_U3A.x_lm258.vlp   1.365E-05
    X_U3A.x_lm258.vln  -8.061E-11

    TOTAL POWER DISSIPATION  -2.90E-01  WATTS



          JOB CONCLUDED

**** 01/24/13 23:13:17 ****** PSpice Lite (October 2012) ****** ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ C:\USERS\MATTHEW EINHORN\DESKTOP\CPL ADC BOARD\Boutput board\Simulation\test-PSpiceFiles\SCHEMATIC1


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .38
